melodies_monet/driver.py

```diff
@@ -653,10 +652,7 @@ def mask_and_scale(self):
                             self.obj[v].data += scale
                         elif d['unit_scale_method'] == '-':
                             self.obj[v].data += -1 * scale
-                    if self.obj[v].units == 'ppv':
-                        print('changing units for {}'.format(v))
-                        self.obj[v].values *= 1e9
-                        self.obj[v].attrs['units'] = 'ppbv'        
```

Comment:
I think removing this is fine, but you might want to add it to your PR description (previously ppv was automatically converted to ppbv for any model...)